doc/call-for-help.rst

```diff
@@ -0,0 +1,160 @@
+################
+Calling for Help
+################
+
+Aside from this troubleshooting guide, there are a number of other sources of documentation and troubleshooting information you can check before submitting an issue, as they might already offer an answer, or at least help you better understand the problem.
+Even if we aren't able to help you, these places might.
+
+
+
+.. rst-class:: fa-offset-h3
+
+========================
+Spyder-related platforms
+========================
+
+.. rst-class:: fasb fa-question-circle
+
+Spyder's FAQ section
+~~~~~~~~~~~~~~~~~~~~
+
+Spyder's documentation has a FAQ section, where you might find the answers that you are looking for.
+
+
+.. rst-class:: fasb fa-globe
+
+Spyder website
+~~~~~~~~~~~~~~
+
+The `Spyder website`_ contains basic information about the IDE and links to many other helpful resources.
+
+.. _Spyder website: https://www.spyder-ide.org/
+
+
+.. rst-class:: fabb fa-google
+
+Spyder Google Group
+~~~~~~~~~~~~~~~~~~~~
+
+Our `Google Group`_ is great for help-related questions, particularly those you aren't sure are a full-on Spyder issue.
+
+.. _Google Group: https://groups.google.com/group/spyderlib
+
+
+.. rst-class:: fabb fa-gitlab
+
+Spyder Gitter
+~~~~~~~~~~~~~
+
+We have a public `Gitter room`_ where you can chat directly with the Spyder developers.
+If you've got a quick question to ask the team and are looking for a quick answer, this is a great place to go.
+
+.. _gitter room: https://gitter.im/spyder-ide/public
+
+
+.. rst-class:: fabb fa-youtube
+
+Spyder's YouTube channel
```

Comment:
I agree the videos are a great resource, especially for beginners. If we move it, I would put it below the website, since the latter is intended as the central starting point for general Spyder needs with links to a variety of resources, vs. the videos are considerably more specific.